### PR TITLE
TableGraph- hover time and crosshairs

### PR DESCRIPTION
**Pull request by ebb-tide**
- [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #1686 

### The problem + the solution
On mouse over table cell: highlight table cell, row and column, and trigger crosshairs on other cells
On mouse over dygraph: trigger crosshairs on other cells, and highlight table cell and row also scroll table to bring highlighted cell in to view.

### CODE REVIEW

File: ui/src/shared/components/Dygraph.js [hunk 1]
```diff
@@ -30,9 +31,9 @@ class Dygraph extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      isSynced: false,
       isHidden: true,
       staticLegendHeight: null,
+      isNotHovering: true,
```
Comment (121watts) [inline ui/src/shared/components/Dygraph.js]: Generally like to keep boolean values in the 'affirmative'.  If we do ever need to negate `!isNotHoevering` reads a bit funny.  No one likes double negatives.

File: ui/src/shared/components/TableGraph.js [hunk 2]
```diff
@@ -52,35 +101,49 @@ class TableGraph extends Component {
     const tableWidth = this.gridContainer ? this.gridContainer.clientWidth : 0
     const tableHeight = this.gridContainer ? this.gridContainer.clientHeight : 0
 
+    const hoverTimeRow =
+      data.length > 1 && hoverTime > 0
+        ? data.findIndex(
+            row => row[0] && _.isNumber(row[0]) && row[0] >= hoverTime
+          )
```
Comment (121watts) [inline ui/src/shared/components/TableGraph.js]: This could probably use its own function

File: ui/src/utils/timeSeriesToDygraph.js [hunk 3]
```diff
@@ -163,10 +171,17 @@ export default function timeSeriesToDygraph(raw = [], isInDataExplorer) {
   }
 }
 
-export const timeSeriesToTable = data => {
-  const {labels, timeSeries} = timeSeriesToDygraph(data, false)
-  const tableData = timeSeries.length
-    ? timeSeries.map(row => row.map(cell => (cell ? cell.toString() : 'null')))
-    : [[]]
-  return {labels, data: [labels, ...tableData]}
+export const timeSeriesToTableGraph = raw => {
```
Comment (121watts) [inline ui/src/utils/timeSeriesToDygraph.js]: Would like to see a test for this update to the `timeSeriesToDygraph` function

Comment (ebb-tide) [reply ui/src/utils/timeSeriesToDygraph.js]: I will do this as a part of the next PR I think we might be changing things around in this function.

**Pull request merged**